tests/e2e/steps_active_set_changes.go

```diff
@@ -86,12 +86,6 @@ func stepsActiveSetChanges() []Step {
 					{Id: ValidatorID("bob"), Stake: 200000000, Allocation: 10000000000},
 					{Id: ValidatorID("carol"), Stake: 700000000, Allocation: 10000000000},
 				},
-				// For consumers that're launching with the provider being on an earlier version
```

Comment:
This test removes the threshold while the ones above have it stil

Reply:
Threshold are deleted from individual `StarConsumerChainAction` and are moved to `startConsumerChain` so we do not have to set the threshold everywhere. This was just part of small refactoring.